modelcontextprotocol/package.json

```diff
@@ -17,7 +17,8 @@
     "lint": "eslint \"./**/*.ts*\"",
     "prettier": "prettier './**/*.{js,ts,md,html,css}' --write",
     "prettier-check": "prettier './**/*.{js,ts,md,html,css}' --check",
-    "test": "jest"
+    "test": "jest",
+    "dxt-pack": "npm run build && dxt pack"
```

Comment:
I think we need to add dxt as a dev dependency?

I would call this build:dxt-extension